extensions/pubsub/standard/document.adoc

```diff
@@ -1,4 +1,4 @@
-= OGC API - Environmental Data Retrieval - Part 2: Publish-Subscribe workflow
+= OGC API - Environmental Data Retrieval - Part 2: Publish-Subscribe Workflow
```

Comment:
`Workflow` or `workflow` (sentence case exept for know terms)?

Reply:
But `Sentence case` not appropriate - it should be `Title Case` ! W not w. ;-)

And Carl says so.